JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -510,6 +510,7 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
     }
 
     JSQMessagesCollectionViewCell *cell = [collectionView dequeueReusableCellWithReuseIdentifier:cellIdentifier forIndexPath:indexPath];
+    cell.accessibilityIdentifier = [NSString stringWithFormat:@"JSQMessagesCollectionViewCellIndexPathRow%ld", (unsigned long)indexPath.row];
```

Comment:
let's just make this `@"(%ld, %ld)", indexPath.section, indexPath.row`

that should be enough...